packages/db/lib/stackable.js

```diff
@@ -22,5 +22,16 @@ class DbStackable extends ServiceStackable {
 
     this.configManager.update(config)
   }
+
+  async getDBInfo () {
+    await this.init()
+    await this.app.ready()
+    if (this.app.platformatic.db) {
+      const connectionInfo = this.app.platformatic.db.connectionInfo
+      const dbschema = this.app.platformatic.dbschema
```

Comment:
Do you need to export all of these data? We would need to implemet this interface for all other types of the services that will have a db. How about exporting a connectionString instead of connectionInfo object?

```js
{
    database: dbname,
     dbSystem: 'postgresql',
     host: '127.0.0.1',
     port: 5432,
     user: 'postgres',
     isPg: true,
     isMySql: undefined,
     isSQLite: undefined,
}
```

Reply:
The same question for the `dbschema`. Do you need the whole object or ony some properties from it?

Reply:
What's the problem in exporting data?

Reply:
The problem is that this is an interface. So if you export all these properties here, we will need to implement the same properties in all other stackables that will have this method.

Reply:
IMHO connection string looks more generic and easier to implement in all other stackables.